nessie/src/test/java/org/apache/iceberg/nessie/BaseTestIceberg.java

```diff
@@ -117,17 +117,21 @@ public void beforeEach(@NessieUri URI nessieUri) throws IOException {
     }
 
     hadoopConfig = new Configuration();
-    catalog = initCatalog(branch);
+    catalog = initCatalog(branch, null);
   }
 
-  NessieCatalog initCatalog(String ref) {
+  NessieCatalog initCatalog(String ref, String hash) {
```

Comment:
maybe just add another `initCatalog` method that accepts a `hash` parameter, so that all the places in the tests don't have to pass `null` all the time